### DISCUSSION TITLE
mermaid vs plantuml (and integration in github)

### DISCUSSION THREAD

**Discussion by berndschatz** [mermaid-js/mermaid#4204] [General]
We are waiting since years for a better integration of plantuml in github,
now i saw this post about mermaid:

https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/

So i assume you work together with microsoft for an integration, and thats good.

But from the view of features (and IMHO also license and technology) mermaid
is far away from plantuml, are there some hionts from your side 
why maybe mermaid is a better choice then plantuml.
I dont expect a pure 100% objectiv comparison from your side ;)

**Comment by Brian-McBride** [comment]
Well, Mermaid runs in the browser and uses SVGs to render the graphics. 
Plantuml requires Java and GraphViz.

That alone breaks them into very different categories. 
It is pretty clear that Marmaid needs to evolve. While an awesome project and it is great so many git sources are supporting it in their UIs, styling and layout options are extremely limited and inconsistent.

**Reply by AlNedorezov** [reply]
FYI, now there is also a version of plantUML that can run in a browser, without the need for a server https://github.com/plantuml/plantuml.js

**Reply by cheoljoo** [reply]
plantuml supports SVG also
